apps/price_pusher/src/logger.ts

```diff
@@ -0,0 +1,35 @@
+import pino from "pino";
+import { Logger, Level } from "pino";
+import { Logger as LoggerInterface } from "ts-log";
+export { Logger } from "pino";
+
+export type LogFormat = "json" | "pretty";
+
+export function createLogger(logLevel?: Level, logFormat?: LogFormat): Logger {
+  return pino({
+    level: logLevel ?? "info",
+    transport:
+      logFormat === "pretty"
+        ? {
+            target: "pino-pretty",
+            options: {
+              colorize: true,
+              ignore: "pid,hostname",
+            },
+          }
+        : undefined,
+  });
+}
+
+export function createPriceServiceConnectionLogger(
+  logger: Logger
+): LoggerInterface {
+  return {
+    trace: logger.trace.bind(logger),
+    debug: logger.debug.bind(logger),
+    // Use debug level here because the info level is too noisy
+    info: logger.debug.bind(logger),
```

Comment:
Instead of doing this, I would recommend simply setting the log level on the child logger to be higher, and maybe exposing an option to pass the log level for price service connection logs.  By grouping the debug and info logs in the same level, you remove the ability for someone to ever be able to selectively separate them.

Presumably you can pass a `pino` instance directly as a `Logger` from `ts-log` since typescript isn't nominal and they are structurally compatible?

Reply:
Makes sense to me. With this change i removed additional complexity of added `logger.ts`